src/centos/7/Dockerfile

```diff
@@ -78,17 +78,23 @@ RUN wget ftp://sourceware.org/pub/binutils/snapshots/binutils-2.29.1.tar.xz && \
     \
     mkdir llvmbuild && \
     cd llvmbuild && \
-    scl enable devtoolset-7 'cmake \
+    scl enable devtoolset-9 'cmake \
         -DCMAKE_BUILD_TYPE=Release \
         -DLLVM_LIBDIR_SUFFIX=64\
         -DLLVM_ENABLE_EH=1 \
         -DLLVM_ENABLE_RTTI=1 \
         -DLLVM_BINUTILS_INCDIR=../binutils-2.29.1/include \
         ../llvm-9.0.0.src' \
     && \
-    scl enable devtoolset-7 'make -j $(($(getconf _NPROCESSORS_ONLN)+1))' && \
+    scl enable devtoolset-9 'make -j $(($(getconf _NPROCESSORS_ONLN)+1))' && \
     make install && \
     cd .. && \
     rm -r llvmbuild && \
     rm -r llvm-9.0.0.src && \
     rm -r binutils-2.29.1
+
+# Add gcc in PATH
+ENV PATH /opt/rh/devtoolset-9/root/usr/bin:$PATH
+
+# Clear yum cache to save ~150 MB
+RUN yum clean all
```

Comment:
I'm not understanding why this is needed.  The yum installation is done way up top and it already runs a clean command at that point.  Why is it necessary at this point?

Reply:
I have reverted this change. I was calculating size impact of container at this point and comparing it what I had when running this container interactively on localhost; didn't realized that we are already doing the cleanup.